pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -596,7 +599,7 @@ class CreateVmModal extends React.Component {
             this.setState({ [key]: value });
             if (this.state.sourceType == PXE_SOURCE) {
                 // If the installation source mode is PXE refresh the list of available networks
-                this.onValueChanged('sourceType', PXE_SOURCE);
+                this.onValueChanged('sourceType', LOCAL_INSTALL_MEDIA_SOURCE);
```

Comment:
I don't understand this change.

Reply:
Ups, I forgot '&& value == LIBVIRT_SESSION_CONNECTION'

Basically when a user switches back to session we need to reset the choice of installation source if before they had chosen PXE, since the selection would not be valid any more.

Reply:
I see. So the comment should rather say "When changing to session connection, reset media source"?